go/ql/src/Security/CWE-770/UncontrolledAllocationSize.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Slice memory allocation with excessive size value
+ * @description Allocating memory for slices with the built-in make function from user-controlled sources
+ *              can lead to a denial of service.
```

Comment:
```suggestion
 * @description Allocating memory for slices with the built-in make function from user-controlled sources can lead to a denial of service.
```

Probably fine on two lines but I don't think I've seen it before, so just in case!

Reply:
Applied, thanks!